lib/rubocop/cop/style/conditional_assignment.rb

```diff
@@ -207,7 +207,7 @@ class ConditionalAssignment < Cop
         MAX = 'Max'.freeze
         SINGLE_LINE_CONDITIONS_ONLY = 'SingleLineConditionsOnly'.freeze
         WIDTH = 'Width'.freeze
-        METHODS = [:[]=, :<<, :=~, :!~, :<=>].freeze
+        METHODS = [:+, :-, :[]=, :<<, :=~, :!~, :<=>].freeze
```

Comment:
I responded to the comments out of order. See below for some thoughts on this.

I will remove this from this PR and focus on it in a separate PR. I am fine with a config option, but I am starting to feel like a new name for this cop would be better.